src/pages/HomePage/ChangePage/changePage.jsx

```diff
@@ -0,0 +1,59 @@
+import MapIcon from '@mui/icons-material/Map';
+import SearchIcon from '@mui/icons-material/Search';
+import { Box, Button, ButtonGroup } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { useContext } from 'react';
+
+import theme from '../../../theme/theme';
+// eslint-disable-next-line import/no-cycle
+import { changePageContext } from '../HomePage';
+import styles from './changePage.module.scss';
+
+const ChangerButton = styled(Button)({
+  flexDirection: 'column',
+  width: '83px',
+  height: '56px',
+  backgroundColor: theme.palette.primary.main,
+  borderRadius: '0px',
+  borderRight: 'none',
+  '&:hover': {
+    backgroundColor: 'none !important',
+  },
+  '&:focus': {
+    backgroundColor: 'none',
+  },
+});
+
+function ChangePage() {
```

Comment:
It's a late suggestion but it's only now that I realised what this component is for. You don't have to change it but changing the name to something like `changeView` might explain the function of this component more readily as `changePage` could suggest it has something to do with changing pages as defined in the `Routes`. Please consider changing the name only if you find some time to do so.